src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/ContainerCleaner.java

```diff
@@ -32,11 +32,15 @@ public void process(String containerName) {
 
         final BlobContainerClient containerClient = storageClient.getBlobContainerClient(containerName);
 
-        envelopeRepository
-            .find(DISPATCHED, false)
-            .forEach(envelope -> {
-                tryToDeleteBlob(envelope, containerClient);
-            });
+        try {
+            envelopeRepository
+                .find(DISPATCHED, false)
+                .forEach(envelope -> {
+                    tryToDeleteBlob(envelope, containerClient);
```

Comment:
try try?

Reply:
External try includes querying the database